packages/cloudscape-react-ts-website/samples/src/App.tsx

```diff
@@ -1,93 +1,144 @@
 /*! Copyright [Amazon.com](http://amazon.com/), Inc. or its affiliates. All Rights Reserved.
 SPDX-License-Identifier: Apache-2.0 */
+import { useCognitoAuthContext } from '@aws-northstar/ui';
+import AppLayout, {
+  AppLayoutProps,
+} from '@aws-northstar/ui/components/AppLayout';
 import {
   BreadcrumbGroup,
   BreadcrumbGroupProps,
   SideNavigation,
   SideNavigationProps,
 } from '@cloudscape-design/components';
-import AppLayout, { AppLayoutProps } from '@cloudscape-design/components/app-layout';
 import { CancelableEventHandler } from '@cloudscape-design/components/internal/events';
-import { createContext, useCallback, useMemo, useState } from 'react';
+import { createContext, useCallback, useEffect, useMemo, useState } from 'react';
 import { Route, Routes, useNavigate } from 'react-router-dom';
-import Auth from './Auth';
 import Config from './config.json';
 import Home from './Home';
-import NavHeader from './NavHeader';
 
 /**
  * Define your nav items here.
  */
 const NAVIGATION_ITEMS: SideNavigationProps.Item[] = [
-  { text: 'home', type: 'link', href: '/' },
+  { text: 'Home', type: 'link', href: '/' },
 ];
 
 /**
  * Context for updating/retrieving the AppLayout.
  */
 export const AppLayoutContext = createContext({
   appLayoutProps: {},
-  setAppLayoutProps: (_: AppLayoutProps) => {},
+  setAppLayoutProps: (_: Partial<AppLayoutProps>) => {},
```

Comment:
NorthStar also has a AppLayoutContext including all the props that can be updated, the consumer can use it directly .